### PR TITLE
databases-week1

### PR DESCRIPTION
**Pull request by patel-prakashkumar**
Hi, I have completed Database week1 Assignment.

### CODE REVIEW

File: databases/week1/Assignment_Task_Queries.sql [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+-- Get the names of all statuses, sorted by the status with most tasks first
+SELECT 
+    status.name, task.title, task.updated
+FROM
+    `task`
+        INNER JOIN
+    `status` ON status.id = task.status_id
+GROUP BY task.title , status.name
+ORDER BY status.name AND task.updated DESC;
```
Comment (Elia-Git) [inline databases/week1/Assignment_Task_Queries.sql]: Use proper indentation rules or avoid indentation. Here you could finish your queries in less than 50 lines where you use 85 lines.'
see the link for how to format your query.
https://learnsql.com/blog/24-rules-sql-code-formatting-standard/

**Review by Elia-Git — COMMENTED** [review]
Make your query lines neat and short for better readability.
Formating. https://learnsql.com/blog/24-rules-sql-code-formatting-standard/

File: databases/week1/Assignment_Task_Queries.sql (hunk 1 above)
Comment (patel-prakashkumar) [reply databases/week1/Assignment_Task_Queries.sql]: Thank you for feedback. I have updated .

Comment (Elia-Git) [inline databases/week1/Assignment_Task_Queries.sql]: Don't see any result from this. What do you think the result will be if you group the tasks `task.title`... probably will turn 35 tasks?
Try this if it makes difference...
```
SELECT 
    status.name, COUNT(status.name)
FROM
    `task`
        INNER JOIN
    `status` ON status.id = task.status_id
GROUP BY status.name
ORDER BY COUNT(status.name)  DESC;
```

File: databases/week1/Assignment_Task_Queries.sql [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+-- Get the single most recently created task
+SELECT 
+    id, title, MAX(created)
+FROM
+    `task`;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT 
+    title, description
```
Comment (Elia-Git) [inline databases/week1/Assignment_Task_Queries.sql]: Here you are asked to return the title & due_date of all tasks that contains "database" in their title or description..
Something like this
```
SELECT
    title, due_date
FROM
    `task`
WHERE
    title LIKE '%database%'
        OR description LIKE '%database%';
```

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls, see the comments given and never ever update your code directly on Github. You are supposed to update/modify your code on VS code & push (commit) it to Github.

File: databases/week1/Assignment_Task_Queries.sql (hunk 2 above)
Comment (patel-prakashkumar) [reply databases/week1/Assignment_Task_Queries.sql]: Yah,  you are right, I have updated

File: databases/week1/Assignment_Task_Queries.sql (hunk 1 above)
Comment (patel-prakashkumar) [reply databases/week1/Assignment_Task_Queries.sql]: I have update task.

**Pull request merged**